metricbeat/module/kubernetes/state_statefulset/_meta/testdata/kube-state-metrics-v1.3.0.plain-expected.json

```diff
@@ -0,0 +1,95 @@
+[
+    {
+        "event": {
+            "dataset": "kubernetes.statefulset",
+            "duration": 115000,
+            "module": "kubernetes"
+        },
+        "kubernetes": {
+            "statefulset": {
+                "_module": {
```

Comment:
If I read the code correct, fields under `_module` should go on the module level. This means this would become `kubernetes.namespace: default`. This would also align well with https://github.com/elastic/beats/blob/master/libbeat/processors/add_kubernetes_metadata/_meta/fields.yml#L21

Please double check with data you receive when running 6.6. The same applies to all the other modules which use this labels magic.